pkg/questionModel/questionModel.go

```diff
@@ -14,73 +27,97 @@ import (
 	"github.com/olekukonko/tablewriter"
 )
 
-const defaultWidth = 20
+const (
+	defaultWidth = 20
+	// Column seperator in glamour tables
+	columnSeperator = "│"
+)
 
 var (
+	// Styling to add left padding to strings
 	noStyle           = lipgloss.NewStyle()
 	xSmallLeftPadding = lipgloss.NewStyle().PaddingLeft(1)
 	smallLeftPadding  = lipgloss.NewStyle().PaddingLeft(3)
 	mediumLeftPadding = lipgloss.NewStyle().PaddingLeft(5)
 	largeLeftPadding  = lipgloss.NewStyle().PaddingLeft(7)
 	xLargeLeftPadding = lipgloss.NewStyle().PaddingLeft(9)
-	focused           = lipgloss.NewStyle().Foreground(lipgloss.Color("170"))
-	blurred           = lipgloss.NewStyle().Foreground(lipgloss.Color("240"))
-	paginationStyle   = list.DefaultStyles().PaginationStyle.PaddingLeft(4)
-	helpStyle         = list.DefaultStyles().HelpStyle.PaddingLeft(4).PaddingBottom(1)
-	quitTextStyle     = lipgloss.NewStyle().Margin(1, 0, 2, 4)
-	errorStyle        = lipgloss.NewStyle().Foreground(lipgloss.Color("9"))
+
+	focused = lipgloss.NewStyle().Foreground(lipgloss.Color("170")) // Pink
+	blurred = lipgloss.NewStyle().Foreground(lipgloss.Color("240")) // Gray
+
+	helpStyle  = list.DefaultStyles().HelpStyle.PaddingLeft(4).PaddingBottom(1)
+	errorStyle = lipgloss.NewStyle().Foreground(lipgloss.Color("9"))
```

Comment:
Thanks for adding the color comments on lines 45 and 46. Can you also add one here?

```suggestion
	errorStyle = lipgloss.NewStyle().Foreground(lipgloss.Color("9")) // Red
```